scripts/format_all.sh

```
@@ -0,0 +1,70 @@
+echo -e "${YELLOW}Formatting problematic files with special attention...${NC}"
+for file in "${PROBLEMATIC_FILES[@]}"; do
+  if [ -f "$file" ]; then
+    format_file "$file"
+  else
+    echo -e "  ${YELLOW}⚠ File not found: $file${NC}"
+  fi
+done
+
+# Format all Dart files in the entire project
+echo -e "${BLUE}Formatting all Dart files in the project...${NC}"
+find . -name "*.dart" -type f | xargs dart format --line-length 80
```

Shellcheck (0.10.0)

[warning] Use 'find .. -print0 | xargs -0 ..' or 'find .. -exec .. +' to allow non-alphanumeric filenames.

(SC2038)